app/Http/Controllers/StatsController.php

```diff
@@ -351,7 +351,7 @@ public function groupsRequirements(): \Illuminate\Contracts\View\Factory|\Illumi
             'user_avg_seedtime' => $user_avg_seedtime,
             'user_account_age'  => $user_account_age,
             'user_seed_size'    => $user_seed_size,
-            'groups'            => Group::orderBy('position')->where('autogroup', 1)->get(),
+            'groups'            => Group::orderBy('position')->where('is_modo', false)->get(),
```

Comment:
It is and was.
IMO it would be nice to also show groups such as Uploader, internal or Trustee. Instead of having this special groups explained somewhere in the FAQ or wiki. But I did not want to show staff groups there (but that's just a personal preference).